components/shared_code/src/shared_data_model/meta/unit.py

```diff
@@ -29,6 +29,7 @@ class Unit(StrEnum):
     SUPPRESSED_VIOLATIONS = "suppressed violations"
     TEST_CASES = "test cases"
     TESTS = "tests"
+    TODOS = "to-do's"
```

Comment:
Is there a better term somehow? Plural should be "to-dos" according to both https://www.merriam-webster.com/dictionary/to-do and https://www.collinsdictionary.com/dictionary/english/to-do .. But however not listed in the Cambridge dictionary

Reply:
I've googled a bit for a more generic term that encapsulates both to-do and fixme. I've seen "task tag" (used in some IDE's apparently) and "codetag" (in a rejected Python PEP), but those are not as self-explanatory as to-do. Maybe "to-do and fix-me comments" is better?

Reply:
Sounds like a great solution. It's very fitting to use "todo" next to "fixme" anyway